crates/pop-common/src/manifest.rs

```diff
@@ -29,33 +29,6 @@ pub fn from_path(path: Option<&Path>) -> Result<Manifest, Error> {
 	Ok(Manifest::from_path(path.canonicalize()?)?)
 }
 
-/// This function is used to determine if a Path is contained inside a workspace, and returns a
-/// PathBuf to the workspace Cargo.toml if found.
-///
-/// # Arguments
-/// * `target_dir` - A directory that may be contained inside a workspace
-pub fn find_workspace_toml(target_dir: &Path) -> Option<PathBuf> {
```

Comment:
As above, since we already have the `find_workspace_toml` code, if it needs a tweak, it’s better to update it or leave it as is and create a separate function specifically for the pallet if needed, rather than remove it and rely on an external library.